minecode/model_utils.py

```diff
@@ -44,7 +44,7 @@
 )
 
 
-def add_package_to_scan_queue(package, pipelines=DEFAULT_PIPELINES, reindex_uri=False, priority=0):
+def add_package_to_scan_queue(package, pipelines=DEFAULT_PIPELINES, reindex_uri=False, priority=100):
```

Comment:
@pombredanne When we add something to the scan queue, do we want it to be placed on top of the stack of packages to be scanned, or should that behavior only be done on requests from the `collect` or similar endpoints?

Reply:
yes, top of the stack when asking for an exact version IMHO

Reply:
ack, tracking this in #502